pkg/ebpf/debugfs.go

```diff
@@ -36,6 +36,21 @@ func GetProbeStats() map[string]int64 {
 	return res
 }
 
+func GetProbeTotals() kprobeStats {
+	stats := kprobeStats{}
+	m, err := readKprobeProfile(KprobeProfile)
```

Comment:
Do we have a sense of how long this takes to run given that we're going to call it every ~30s now?

Reply:
We already run this on a  5 second loop for expvars.

Reply:
It isn't super precise, but the overhead seems neglible. If you want me to write a proper Go benchmark, let me know.
```
$ time -v cat /sys/kernel/debug/tracing/kprobe_profile
...
Command being timed: "cat /sys/kernel/debug/tracing/kprobe_profile"
	User time (seconds): 0.00
	System time (seconds): 0.00
	Percent of CPU this job got: 0%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 0:00.00
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 1688
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 0
	Minor (reclaiming a frame) page faults: 73
	Voluntary context switches: 1
	Involuntary context switches: 10
	Swaps: 0
	File system inputs: 0
	File system outputs: 0
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 0
```

Reply:
Nope - sounds good 👍